src/main/ui/src/components/header/StateDropdown.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Nav from "react-bootstrap/Nav";
+import NavDropdown from "react-bootstrap/NavDropdown";
+import { states } from "../../data";
+
+export default function StateDropdown() {
+	return (
+		<Nav>
+			<NavDropdown id="nav-dropdown" title="View by State" menuVariant="dark">
+				{states.map((item) => (
+					<NavDropdown.Item href="" key={item.id}>
```

Comment:
`... href="" key=index`

adding the index in the map lets you use the maps index as a key and eliminates the error in the console when you run the code. Before making the change `npm start` look in the console in the browser and you should see a big long error that points to this file. 

Make the changes and check the console again and it should be fixed. It's a best practice to check the console and see if it's complaining about anything. Def a good habit :)

As usual though your code is great. Fix this one thing and we can merge it! Slack me if you have any questions please.